torch/ao/quantization/fx/prepare.py

```diff
@@ -433,7 +435,13 @@ def get_arg_target_dtype_as_output(
     argument in quantized graph will match what is specified by the qconfig
     """
     assert isinstance(arg, Node)
-    if is_activation_post_process_node(arg, modules):
+    # Custom module LSTM output is a tuple that we broke down into the internal nodes in order
+    # to insert DeQuantStubs (see `_insert_dequant_stubs_for_custom_module_lstm_output`)
+    # Here we must trace through that subgraph to reach the original LSTM node
```

Comment:
I think changing "trace through ..." to "trace back from the args to reach ..." might be more accurate